src/app/api/services/crud/crud.service.ts

```diff
@@ -59,4 +59,19 @@ export class CrudService {
             url: `${baseUrl}/id/${id}`
         }).pluck('entity');
     }
+
+    /**
+     * Delete item by id from the data loaded
+     *
+     * @param {string} baseUrl
+     * @param {string} id
+     * @returns {Observable<any>}
+     * @memberof CrudService
+     */
+    deleteDataById(baseUrl: string, id: string): Observable<any> {
```

Comment:
I think this method should be called just ```delete``` because this is a "crud" service I was talking to @freddyucv about this, ask him for more info.